src/components/GetUniversalSearchResults.component.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
```

Comment:
since this component and the `GetStorySearchResults` component are so similar, we should make a general `POISearch` component that acts as a wrapper around the `AutoSuggest` component that takes in props that would define how the story search/universal search would differ.